install/updates/preconfig/0.9/preconfig_0.9.inc.php

```diff
@@ -708,4 +708,16 @@ function parseAndOutputPreconfig(&$has_preconfig, &$return, $current_version, $c
 	    $question.= makeyesno('enable_letsencrypt', '1', '0', '1').'<br />';
 	    eval("\$return.=\"" . getTemplate("update/preconfigitem") . "\";");
 	}
+	
+	if(versionInUpdate($current_db_version, '201603150')) {
+	    $has_preconfig = true;
+	    $description  = 'You can chose which TLS versions will be enabled for your webservers SSL setup.<br />Please select the desired TLS versions below.<br /><br />';
+	    $question = '<strong>Do you want to enable Let\'s Encrypt? (default: yes):</strong>&nbsp;';
+	    $question .= '<select name="update_ssl_protocols">';
+	    $options = makeoption('TLSv1', 'TLSv1', 'TLSv1');
```

Comment:
The default select is not a "multiple" select, so setting each option as selected would end up in only one option being selected by default.

This means the select should look like this:
`<select name="update_ssl_protocols[]" multiple="multiple">`

which then leads to a different handling of the POST-value in https://github.com/Froxlor/Froxlor/pull/348/files#diff-29773a1587adda5af698b968f2130e49R3301 (it will be an array, not just a string)

Also, you are inserting the new setting using Settings::AddNew in https://github.com/Froxlor/Froxlor/pull/348/files#diff-29773a1587adda5af698b968f2130e49R3298 but then SQL-Insert the values from the preconfig manually AGAIN to the settings (without using the Settings::AddNew() function)  @ https://github.com/Froxlor/Froxlor/pull/348/files#diff-29773a1587adda5af698b968f2130e49R3302 - please fix (just look and earlier code how we do it)